src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,94 @@
+.menu {
+  background-color: $colorGray;
```

Comment:
This isn't the same color with the design. But it looks better imo

Reply:
Oh haha I picked the wrong color.  🤦🏻‍♀ Fixed it now.